server/events/runtime/init_step_runner.go

```diff
@@ -16,12 +16,12 @@ func (i *InitStepRunner) Run(ctx models.ProjectCommandContext, extraArgs []strin
 	if ctx.ProjectConfig != nil && ctx.ProjectConfig.TerraformVersion != nil {
 		tfVersion = ctx.ProjectConfig.TerraformVersion
 	}
-	terraformInitCmd := append([]string{"init", "-input=false", "-no-color"}, extraArgs...)
+	terraformInitCmd := append([]string{"init", "-input=false", "-no-color", "-upgrade"}, extraArgs...)
```

Comment:
I'm not sure, but do you need to do `-upgrade=true` instead?

Reply:
Hey Luke, thanks for the quick feedback. Was looking at this TF doc: https://www.terraform.io/docs/commands/init.html#upgrade

Reply:
It is also referenced here as just -upgrade:
https://www.hashicorp.com/blog/hashicorp-terraform-0-11

Reply:
but can add the =true if you want to be extra careful?

Reply:
Nope, looks good!